static/js/tools/map/title.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Title component before a chart is shown.
+ */
+
+import React, { useContext } from "react";
+
+import { Context } from "./context";
+import { ifShowChart } from "./util";
+
+export function Title(): JSX.Element {
```

Comment:
I wonder if instead of having a separate title component and info component, we could just have one LandingPage component that returns null if showChart is true? And then have another ChartPage component that returns null if showChart is false? Then App would render both those components and PlaceSelector can go into both those components.

Reply:
Done

Reply:
ah, ok then

Reply:
We may come back to this later. I also feel there could be a better way to organize it.